docs/pages/reference/authentication.mdx

```diff
@@ -117,14 +117,13 @@ $ tctl create -f cap.yaml
 </TabItem>
 </Tabs>
 
-### Local user login failure rules
+### Local user policies
 
-A local user is blocked from attempting logins if, within a 30 minute window, a local user has multiple:
+Teleport requires that passwords for local users are at least 12 characters long.
```

Comment:
```suggestion
Teleport requires that passwords for local users be at least 12 characters long.
```

Tiny nit if we're using the [subjunctive mood](https://www.merriam-webster.com/grammar/getting-in-the-subjunctive-mood).